src/lib/annotations/doc/DocHighlightDialog.js

```diff
@@ -100,6 +100,33 @@ class DocHighlightDialog extends AnnotationDialog {
         this.toggleHighlight();
     }
 
+    /** TEMPORARY override to hide or show UI based on enabled annotation types.
+     *
+     * @param {boolean} [showPlain] - Whether or not show plain highlight UI
+     * @param {boolean} [showComment] - Whether or not show comment highlight UI
+     * @return {void}
+     */
+    show(showPlain = true, showComment = true) {
+        const plainButtonEl = this.highlightDialogEl.querySelector(`button.${constants.CLASS_ADD_HIGHLIGHT_BTN}`);
+        const commentButtonEl = this.highlightDialogEl.querySelector(
+            `button.${constants.CLASS_ADD_HIGHLIGHT_COMMENT_BTN}`
+        );
+
+        if (showPlain) {
+            annotatorUtil.showElement(plainButtonEl);
+        } else {
+            annotatorUtil.hideElement(plainButtonEl);
```

Comment:
You wouldn't ever show the plain highlight dialog AND the comments dialog correct? Can we combine these 2 if/else statements?

Reply:
Yes we would, as that's how it currently is. This code is for only showing/hiding the buttons

Reply:
ah right not showing/hiding specific types of dialogs. got it